### DISCUSSION TITLE
Quit to Workbench button in Amiga Test Kit

### DISCUSSION THREAD

**Discussion by ghost** [keirf/Amiga-Stuff#63] [Ideas]
Dear Keir,

I use ATK also on my Amiga workbench harddrive. Could it be possible to make an option/button in ATK that allows quit to workbench without resetting the Amiga? So that we have both choices of reset or quit.

Thank you for your continuous support for ATK, it's very usefull.

Kind regards,
Hans

**Comment by keirf** [comment]
No. Short answer haha.

ATK does not respect OS memory layout so OS gets trampled. It would be quite some work to fix this. And perhaps impossible for the memory tests. So it won't happen.
